bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/common/ProductListItem.java

```diff
@@ -51,6 +51,16 @@ default String getImageURL() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns the product image alt text of this {@code ProductListItem}.
+     *
+     * @return the product image alt text of this list item or {@code null}
+     */
+    @Nullable
+    default String getImageAlt() {
+        throw new UnsupportedOperationException();
```

Comment:
I think we stopped doing this a while ago.

Reply:
I followed the other methods if the interface, but I'm also ok to remove it. Maybe we should clean up all methods and other classes?

Reply:
+1 for cleanup. 
There was already a cleanup in #199. A quick search shows that this is the only Sling model interface with such methods but there are new occurrences in datalayer like CategoryData, ProductData. @laurentiumagureanu , is that OK to clean them up as well?

Reply:
Removed defaults for the entire ProductListItem interface.